playbooks/roles/airship-configure-caasp/tasks/main.yml

```diff
@@ -16,8 +16,11 @@
 
 - name: create namespaces in CaaSP
   command: "kubectl apply -f /tmp/namespaces.yaml --overwrite=true"
+  register: create_namespaces_result
+  changed_when: "create_namespaces_result.stdout|search('created')"
   tags:
     - run
+    - skip_ansible_lint
```

Comment:
Why is there a ``skip_ansible_lint``, if you have a changed_when focusing on that use case? What was the issue?

Reply:
thx  @evrardjp  I should have removed 'skip_ansible_lint' tag, I am not getting any error when I remove it. Will handle this as a new PR